lib/projectPlatformVersion.js

```diff
@@ -0,0 +1,16 @@
+const { fetchPlatformVersions } = require('../api/developerProjects');
+
+const fetchDefaultVersion = async accountId => {
+  const platformVersions = await fetchPlatformVersions(accountId);
+  return platformVersions.defaultPlatformVersion;
+};
+
+const fetchActiveVersions = async accountId => {
+  const platformVersions = await fetchPlatformVersions(accountId);
+  return platformVersions.activePlatformVersions;
+};
```

Comment:
I don't have tons of context on this issue, but do we need this function? Doesn't look like its being used anywhere

Reply:
We don't technically need this function yet, but I wanted to include it for future iterations. I'll remove it for now 👍